samples/bmi/calculator/src/main/java/io/servicecomb/samples/bmi/CalculatorServiceImpl.java

```diff
@@ -30,16 +38,34 @@
    * {@inheritDoc}
    */
   @Override
-  public double calculate(double height, double weight) {
+  public Map<String, String> calculate(double height, double weight) {
     if (height <= 0 || weight <= 0) {
       throw new IllegalArgumentException("Arguments must be above 0");
     }
     double heightInMeter = height / 100;
     double bmi = weight / (heightInMeter * heightInMeter);
-    return roundToOnePrecision(bmi);
+    
+    double result = roundToOnePrecision(bmi);
+
+    MicroserviceInstance instance = RegistryUtils.getMicroserviceInstance();
+    String processId = instance.getInstanceId().substring(0, 12);  
+    
+    Date date = new Date(); 
+    DateFormat format = new SimpleDateFormat("HH:mm:ss");
+    String callTime = format.format(date);
+
+    Map<String, String> resultMap = new HashMap<String, String>();
```

Comment:
how can consumers know there are result/processId/callTime in response message?

Reply:
We can create a response class which holds the result , processId and callTime for the service consumer to use.